### PR TITLE
Module changes

### PR DESCRIPTION
**Pull request by jwharm**
Two changes:
1. I changed the module name of glib from `org.glib` to `org.gtk.glib`. (`org.glib`, as a reverse domain name, is not owned by the glib developers)
2. I split Adwaita into a separate module. It is distributed separately from Gtk so I think this is better. I had to add a `dependsOn` directive to the `javadoc` tasks because Gradle tried to generate the Adwaita javadoc before GLib and Gtk. I'm not sure if this is the correct way to do this though.

### CODE REVIEW

File: gstreamer/build.gradle.kts [hunk 1]
```diff
@@ -91,5 +67,6 @@ setupGenSources {
 }
 
 tasks.javadoc {
+    dependsOn(project(":glib").tasks.javadoc)
```
Comment (JFronny) [inline gstreamer/build.gradle.kts:70]: Ideally, this dependency should be handled by linksOffline.
Maybe our existing extension method could be replaced by an extension method for Javadoc?
Could probably be implemented as follows:
```kt
fun Javadoc.linksOffline(url: String, project: Project) {
    if (options is StandardJavadocDocletOptions) {
        val task = project.tasks.named<Javadoc>("javadoc").get()
        (options as StandardJavadocDocletOptions).linksOffline(url, task.destinationDir.toString())
        dependsOn(task)
    } else throw TypeCastException("Unexpected javadoc options type")
}
```

**Review by JFronny — APPROVED** [review]
It seems like a good idea to seperate libadwaita from the gtk4 bindings, especially now that the generator allows seamless cross-references anyway.
The only thing I don't like about this is the detail with linksOffline, but that should be an easy fix.

File: gstreamer/build.gradle.kts (hunk 1 above)
Comment (jwharm) [reply gstreamer/build.gradle.kts]: That worked perfectly. (commit ffcd150)

**Comment by jwharm** [comment]
I changed the package name of GStreamer to use the correct domain (`org.freedesktop.gstreamer`).

@JFronny According to the Gtk developers on Matrix chat, the GLib and GTK packages should actually be under `org.gnome` and not `org.gtk`.
I want to change this sometime soon. This will break current API, but will also resolve the ugly situation with the current `org.gtk.gtk` package (which will become `org.gnome.gtk`, much nicer in my opinion).

**Pull request merged**